man/systemd.netdev.xml

```diff
@@ -315,6 +315,26 @@
           </listitem>
         </varlistentry>
         <varlistentry>
+          <term><varname>AgeingTimeSec=</varname></term>
+          <listitem>
+            <para>AgeingTimeSec specifies the number of seconds a MAC Address will be kept in
```

Comment:
sorry I didn't notice this earlier, but please if you mention a setting in the man page, suffix it with `=` and enclose it in `<varname>` so that it always appears the same. That said, given that this is the explanation for field and you mention its name immediately before, please just change this to "This" or so instead. (i.e. "This specifies the number…")